README.md

```diff
@@ -6,38 +6,81 @@ the Transmission BitTorrent client, using its HTTP RPC protocol.
 
 # DEPENDENCIES
 
-The following packages are required dependencies (debian/ubuntu).
+### Runtime Required Dependencies
+The following packages are required dependencies:
 
-```bash
-libgtk-3-dev automake autoconf gcc libgeoip-dev gettext autoconf-archive libappstream-dev appstream-util libcurl4-openssl-dev libjson-glib-dev
+ - gtk >= 3.16
+ - glib >= 2.44 (including gio and gthread)
+ - json-glib >= 0.8
+ - libcurl
+ - GNU gettext
+
+An example command for getting the packages required for an Ubuntu/Debian system:
+
+```
+# apt-get install libgtk-3-dev libgeoip-dev gettext libcurl4-openssl-dev libjson-glib-dev
 ```
 
-# BUILDING
+### Runtime Optional Dependencies
+The following packages are optinal dependencies
+
+ - libmrss >= 0.18, RSS feed support
+ - libproxy, HTTP proxy support
+ - libgeoip, country of origin of peers
+ - libnotify, desktop notifications support
+ - libappindicator, Application tray support
 
-Optionally install `libgeoip` and its headers to see the country
-origin of peers.
+### Build Dependencies
+Transmission Remote Gtk uses meson for its build system. A relatively new version of
+meson is required.
 
+ - meson >= 0.59.0
+ - ninja
+ - gcc >= 4.5
+ - appstream-util (optional, desktop/appdata validation)
+ - pod2man (optional, manpage generation)
 
-##  Building a transmission-remote-gtk release from the command line:
+An example of getting the build dependencies for an Ubuntu/Debian system:
 
-```bash
-$ tar zxfv transmission-remote-gtk-1.2.tar.gz
-$ cd transmission-remote-gtk-1.2
-$ ./configure
-$ make
-# make install
+```
+# apt-get install gcc python3 meson ninja-build libappstream-dev appstream-util perl
```

Comment:
```suggestion
# apt install gcc meson appstream-util perl
```
Seems more accurate. Meson ofc depends on ninja/python3.

Reply:
This however is missing glib/gtk and would clearly fail

Reply:
Changed definition of required dependencies to reflect this